org.civicrm.reqangular/src/common/decorators/ui-bootstrap/uib-modal.decorator.js

```diff
@@ -20,38 +20,23 @@ define([
     function open () {
       var modalInstance;
       var elements = [
```

Comment:
Are you sure this solution works in IE11? These lines that are being deleted were added to fix a specific IE11 issue:
https://github.com/civicrm/civihr/pull/2400

Reply:
This solution is now pure CSS, so no more replaces, no more hardcoded attributes.

Reply:
![image](https://user-images.githubusercontent.com/3973243/39424261-a99ceab2-4c6d-11e8-8682-52c1c0241d83.png)
left a comment to QA ^ it works for me in IE11